File: code/maket.cpp
#include "iostream"
using namespace std;

struct Img
{
    int image[1024][768];
};

class Robot
{
private:
    int coordinat;
public:
    void set_coordinat(int coor){coordinat = coor;}
    void calculate_way(int coordinat){}
    void move(int traject){}
    void clear(){}
};

class Camera
{
public:
    Img get_img();
};

class System
{
private:
    Robot rob;
    Camera cam;
public:
    void detect()
    {
        cam.get_img();
    }
    void calculate_coordinates(Img img){}
    void transfer_coordinates(int coordinat)
    {
        rob.set_coordinat(coordinat);
    }

};

int main()
{
    System sys;
    return 0;
}

File: code/maket.py
class Robot:
    __cordinat = None
    def __init__(self) -> None:
        print("Robot start...")
    def set_cordinat(cord):
        __cordinat = cord
    def calculat_way():
        pass
    def move():
        pass
    def clear():
        pass

class Camera:
    def __init__(self) -> None:
        print("Camera start...")
    def get_img():
        pass

class System:
    
    def __init__(self) -> None:
        print("System start...")
        self.robot = Robot()
        self.camera = Camera()
    def detect():
        pass
    def calculate_coordinates():
        pass
    def transfer_coordinates():
        pass

sys = System()